types/module/module.go

```diff
@@ -322,4 +340,29 @@ func (m *Manager) EndBlock(ctx sdk.Context, req abci.RequestEndBlock) abci.Respo
 	}
 }
 
-// DONTCOVER
+//____________________________________________________________________________
```

Comment:
Not blocking, but from my exp, I find that if you find yourself using this comment separator, it's probably a good indication to move this into it's own file (i.e. `module/simulation.go`)